src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
```

Comment:
I just tried closing windows/quitting the app after running `sudo chown -R ~/.atom/storage`. I see `Uncaught Error: EACCES: permission denied` in the console, but otherwise everything works as expected.

So I don't have a strong opinion about it, but generally I err on the side of not swallowing errors. Does this case warrant it?